Violating generation throws exception when run with two rules

## Bug Report

### Steps to Reproduce:

- Create profile with two rules e.g. : 
```
{
    "schemaVersion": "0.1",
    "description": "Testing Profile",
    "fields": [
        {
            "name": "foo"
        }
    ],
    "rules": [
        {
            "rule": "rule1",
            "constraints": [
                {
                    "field": "foo",
                    "is": "equalTo",
                    "value": "goodbye"
                }
            ]
        },
        {
            "rule": "rule2",
            "constraints": [
                {
                        "field": "foo",
                        "is": "ofType",
                        "value": "string"
                }
            ]
        }
    ]
}
```

- Run generator in violating mode

### Expected Result:
Generator should output two csv files with violating data

### Actual Result:
Generator produces only the first file, fails with exception:
```
Exception in thread "main" picocli.CommandLine$ExecutionException: Error while running command (com.scottlogic.deg.generator.commandline.GenerateCommandLine@72ade7e3): java.lang.IllegalStateException: Timer already cancelled.
	at picocli.CommandLine.execute(CommandLine.java:1051)
	at picocli.CommandLine.access$900(CommandLine.java:142)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:1246)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:1214)
	at picocli.CommandLine$AbstractParseResultHandler.handleParseResult(CommandLine.java:1122)
	at picocli.CommandLine.parseWithHandlers(CommandLine.java:1405)
	at picocli.CommandLine.parseWithHandler(CommandLine.java:1344)
	at com.scottlogic.deg.generator.App.main(App.java:18)
Caused by: java.lang.IllegalStateException: Timer already cancelled.
	at java.util.Timer.sched(Timer.java:397)
	at java.util.Timer.scheduleAtFixedRate(Timer.java:328)
	at com.scottlogic.deg.generator.generation.VelocityMonitor.generationStarting(VelocityMonitor.java:41)
	at com.scottlogic.deg.generator.generation.DecisionTreeDataGenerator.generateData(DecisionTreeDataGenerator.java:74)
	at com.scottlogic.deg.generator.StandardGenerationEngine.generateDataSet(StandardGenerationEngine.java:32)
	at com.scottlogic.deg.generator.violations.ViolationGenerationEngine.generateDataSet(ViolationGenerationEngine.java:40)
	at com.scottlogic.deg.generator.GenerateExecute.run(GenerateExecute.java:65)
	at com.scottlogic.deg.generator.commandline.CommandLineBase.run(CommandLineBase.java:48)
	at picocli.CommandLine.execute(CommandLine.java:1043)
	... 7 more
```

--- a/generator/src/main/java/com/scottlogic/deg/generator/generation/VelocityMonitor.java
+++ b/generator/src/main/java/com/scottlogic/deg/generator/generation/VelocityMonitor.java
@@ -24,7 +24,7 @@ public class VelocityMonitor implements ReductiveDataGeneratorMonitor {
     private OffsetDateTime startedGenerating;
     private long rowsSinceLastSample;
     private BigInteger rowsEmitted;
-    private Timer timer = new Timer(true);
+    private Timer timer;
     private DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HH:mm:ss");
     private long previousVelocity = 0;
 
@@ -38,6 +38,7 @@ public class VelocityMonitor implements ReductiveDataGeneratorMonitor {
         System.out.println("Number of rows | Velocity (rows/sec) | Velocity trend");
         System.out.println("---------------+---------------------+---------------");
 
+        timer = new Timer(true);
         timer.scheduleAtFixedRate(new TimerTask() {
             @Override
             public void run() {

